tests/Integration/PrestaShopBundle/Controller/Admin/Improve/Design/PositionsControllerTest.php

```diff
@@ -43,10 +67,14 @@ public function testUnhooksListAction()
             ),
             [
                 'unhooks' => [
-                    '41_1',
-                    '65_1000',
-                    '1000000_3',
-                    '65_3',
+                    sprintf('%d_%d', $this->hookId, $this->moduleId),
```

Comment:
wow so in the previous test the `41` value didn't even match the `65` id?

Reply:
It was because there were some modules in database, to prevent this I load specific modules store in `tests/resources/modules` instead.

Reply:
ok cleaner